apps/web/src/pages/CommunityInterests/CommunityTalent/CommunityTalentTable.tsx

```diff
@@ -0,0 +1,251 @@
+import { useState } from "react";
+import {
+  ColumnDef,
+  PaginationState,
+  SortingState,
+  createColumnHelper,
+} from "@tanstack/react-table";
+import { useIntl } from "react-intl";
+import { useQuery } from "urql";
+
+import { notEmpty } from "@gc-digital-talent/helpers";
+import {
+  graphql,
+  CommunityTalentTableQuery as CommunityTalentTableQueryType,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+import Table from "~/components/Table/ResponsiveTable/ResponsiveTable";
+import { INITIAL_STATE } from "~/components/Table/ResponsiveTable/constants";
+import { getFullNameLabel } from "~/utils/nameUtils";
+import useRoutes from "~/hooks/useRoutes";
+import cells from "~/components/Table/cells";
+import adminMessages from "~/messages/adminMessages";
+import processMessages from "~/messages/processMessages";
+
+import {
+  classificationAccessor,
+  interestAccessor,
+  transformSortStateToOrderByClause,
+  usernameCell,
+} from "./utils";
+
+const CommunityTalentTable_Query = graphql(/* GraphQL */ `
+  query CommunityTalentTable(
+    $where: CommunityInterestFilterInput
+    $first: Int
+    $page: Int
+    $orderBy: [QueryCommunityInterestsPaginatedOrderByRelationOrderByClause!]
+  ) {
+    communityInterestsPaginated(
+      where: $where
+      first: $first
+      page: $page
+      orderBy: $orderBy
+    ) {
+      data {
+        id
+        jobInterest
+        trainingInterest
+        user {
+          id
+          firstName
+          lastName
+          workEmail
+          preferredLang {
+            label {
+              localized
+            }
+          }
+          lookingForEnglish
+          lookingForFrench
+          lookingForBilingual
+          currentClassification {
+            group
+            level
+          }
+        }
+        community {
+          name {
+            localized
+          }
+        }
+      }
+      paginatorInfo {
+        count
+        currentPage
+        firstItem
+        hasMorePages
+        lastItem
+        lastPage
+        perPage
+        total
+      }
+    }
+  }
+`);
+
+type CommunityTalentTableQueryCommunityInterestType =
+  CommunityTalentTableQueryType["communityInterestsPaginated"]["data"][number];
+
+const columnHelper =
+  createColumnHelper<CommunityTalentTableQueryCommunityInterestType>();
```

Comment:
Is there a reason we did not just use a fragment here?

`createColumnHelper<CommunityTalentTableRowFragment>`

Reply:
Not really, I think we've just been a bit inconsistent with these things. 
I have changed it around in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12733/commits/fbf907585b88e6ae77f90149b936acd1eb0a044c